src/sentry/static/sentry/app/views/projectSavedSearches.jsx

```diff
@@ -210,6 +220,10 @@ class ProjectSavedSearches extends AsyncView {
   }
 
   renderBody() {
+    if (this.hasOrgSavedSearch()) {
```

Comment:
nit: Might be slightly cleaner to have a parent component that does:
```
return this.context.organization.features.includes('org-saved-search') ? <ProjectSavedSearches /> : <NotFound>
```

Reply:
This component's parent is the Route. Would it be reasonable to add a container component for this logic?

Reply:
Is there a reason that it'd be unreasonable given the Route parent? (I agree it's cleaner, but probably doesn't matter too much since this view will be going away?)

Reply:
Ok. I can add a container component. It gives a cleaner implementation while this code lives.